# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/moz.build
# Commit: 60d215858649
# Full Hash: 60d215858649ce5f058db2f715004749c17e7c93
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1423253
# File Overlap Count: 1
# Description:
#   Bug 1423253 - Add a drift compensator to reclock video timestamps. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D22902
# ==============================================================================

diff -r 54b4c9399b81 -r 60d215858649 dom/media/gtest/moz.build
--- a/dom/media/gtest/moz.build	Fri Mar 22 11:42:23 2019 +0000
+++ b/dom/media/gtest/moz.build	Fri Mar 22 11:42:31 2019 +0000
@@ -25,6 +25,7 @@
     'TestBlankVideoDataCreator.cpp',
     'TestCDMStorage.cpp',
     'TestDataMutex.cpp',
+    'TestDriftCompensation.cpp',
     'TestGMPCrossOrigin.cpp',
     'TestGMPRemoveAndDelete.cpp',
     'TestGMPUtils.cpp',